charts/logging-apps/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: logging-apps
 description: Argo CD app-of-apps config for logging applications
 type: application
 # version and appVersion are in sync in this chart!
-version: 0.0.2
-appVersion: 0.0.1
+version: 0.1.0
```

Comment:
@tongpu and I were discussing this last week. 

Do we need to do a minor release bump when ADDING a new app? Or would a patchversion bump suffice? What are your thoughts @hairmare ?

Reply:
As per semver, bumping the minor version indicates that a new feature was added. A patch bump would indicate that we are fixing an issue.

Reply:
Okay this makes totally sense.